eta/core/serial.py

```diff
@@ -519,7 +519,8 @@ def __contains__(self, key):
         return key in self.__elements__
 
     def __iter__(self):
-        return itervalues(self.__elements__)
+        for v in itervalues(self.__elements__):
```

Comment:
hmm I guess it was bad of me to return an `iterable`, not actually an `iterator`...

You can write this more directly as:
```py
def __iter__(self):
    return iter(itervalues(self.__elements__))
```

Reply:
Oh sure, I thought that would look worse for some reason, but that works.